tests/scene/test_style_box_flat.h

```diff
@@ -0,0 +1,284 @@
+	style_box_flat->set_corner_radius(CORNER_TOP_LEFT, 10);
+	style_box_flat->set_corner_radius(CORNER_BOTTOM_RIGHT, 15);
+	style_box_flat->set_corner_radius(CORNER_BOTTOM_LEFT, 20);
+
+	CHECK(style_box_flat->get_corner_radius(CORNER_TOP_RIGHT) == 5);
+	CHECK(style_box_flat->get_corner_radius(CORNER_TOP_LEFT) == 10);
+	CHECK(style_box_flat->get_corner_radius(CORNER_BOTTOM_RIGHT) == 15);
+	CHECK(style_box_flat->get_corner_radius(CORNER_BOTTOM_LEFT) == 20);
+}
+
+TEST_CASE("[TestStyleBoxFlat] set_corner_detail, get_corner_detail") {
+	Ref<StyleBoxFlat> style_box_flat = memnew(StyleBoxFlat);
+
+	style_box_flat->set_corner_detail(5);
+	CHECK(style_box_flat->get_corner_detail() == 5);
+
+	style_box_flat->set_corner_detail(0);
+	CHECK(style_box_flat->get_corner_detail() == 1);
+
+	style_box_flat->set_corner_detail(-1);
+	CHECK(style_box_flat->get_corner_detail() == 1);
+
+	style_box_flat->set_corner_detail(25);
+	CHECK(style_box_flat->get_corner_detail() == 20);
+}
+
+TEST_CASE("[TestStyleBoxFlat] set_expand_margin_all, get_expand_margin") {
+	Ref<StyleBoxFlat> style_box_flat = memnew(StyleBoxFlat);
+
+	style_box_flat->set_expand_margin_all(25);
+
+	CHECK(style_box_flat->get_expand_margin(SIDE_RIGHT) == 25);
+	CHECK(style_box_flat->get_expand_margin(SIDE_LEFT) == 25);
+	CHECK(style_box_flat->get_expand_margin(SIDE_BOTTOM) == 25);
+	CHECK(style_box_flat->get_expand_margin(SIDE_TOP) == 25);
+}
+
+TEST_CASE("[TestStyleBoxFlat] set_expand_margin_individual, get_expand_margin") {
+	Ref<StyleBoxFlat> style_box_flat = memnew(StyleBoxFlat);
+
+	style_box_flat->set_expand_margin_individual(5, 10, 15, 20);
+
+	CHECK(style_box_flat->get_expand_margin(SIDE_LEFT) == 5);
+	CHECK(style_box_flat->get_expand_margin(SIDE_TOP) == 10);
+	CHECK(style_box_flat->get_expand_margin(SIDE_RIGHT) == 15);
+	CHECK(style_box_flat->get_expand_margin(SIDE_BOTTOM) == 20);
+}
+
+TEST_CASE("[TestStyleBoxFlat] set_expand_margin, get_expand_margin") {
+	Ref<StyleBoxFlat> style_box_flat = memnew(StyleBoxFlat);
+
+	style_box_flat->set_expand_margin(SIDE_RIGHT, 5);
+	style_box_flat->set_expand_margin(SIDE_LEFT, 10);
+	style_box_flat->set_expand_margin(SIDE_BOTTOM, 15);
+	style_box_flat->set_expand_margin(SIDE_TOP, 20);
+
+	CHECK(style_box_flat->get_expand_margin(SIDE_RIGHT) == 5);
+	CHECK(style_box_flat->get_expand_margin(SIDE_LEFT) == 10);
+	CHECK(style_box_flat->get_expand_margin(SIDE_BOTTOM) == 15);
+	CHECK(style_box_flat->get_expand_margin(SIDE_TOP) == 20);
+}
+
+TEST_CASE("[TestStyleBoxFlat] set_draw_center, is_draw_center_enabled") {
+	Ref<StyleBoxFlat> style_box_flat = memnew(StyleBoxFlat);
+
+	style_box_flat->set_draw_center(true);
+
+	CHECK(style_box_flat->is_draw_center_enabled() == true);
+}
+
+TEST_CASE("[TestStyleBoxFlat] set_skew, get_skew") {
+	Ref<StyleBoxFlat> style_box_flat = memnew(StyleBoxFlat);
+
+	Vector2 skewValue(1.0, 2.0);
+
+	style_box_flat->set_skew(skewValue);
+
+	CHECK(style_box_flat->get_skew() == skewValue); // Will this work?
```

Comment:
Remove redundant comment:

```suggestion
	CHECK(style_box_flat->get_skew() == skewValue);
```